lib/project_types/theme/commands/serve.rb

```diff
@@ -6,11 +6,22 @@ class Serve < ShopifyCli::Command
 
       options do |parser, flags|
         parser.on('--env=ENV') { |env| flags[:env] = env }
+        parser.on('--allow-live') { flags['allow-live'] = true }
+        parser.on('--notify') { flags['notify'] = true }
       end
 
       def call(*)
+        if options.flags[:env]
+          env = options.flags[:env]
+          options.flags.delete(:env)
+        end
+
+        flags = options.flags.map do |key, _value|
```

Comment:
This is a somewhat simple piece of code, but it could be useful for other commands as well - should we centralize it somewhere?